src/Kdyby/Google/exceptions.php

```diff
@@ -22,6 +22,111 @@
 }
 
 
+/**
+ * Thrown when an API call returns an exception.
+ *
+ * @author Naitik Shah <naitik@facebook.com>
+ */
+class GoogleApiException extends \Exception implements Exception
+{
+
+	/**
+	 * The result from the API server that represents the exception information.
+	 */
+	protected $result;
+
+
+
+	/**
+	 * Make a new API Exception with the given result.
+	 *
+	 * @param array $result The result from the API server
+	 */
+	public function __construct($result)
+	{
+		$this->result = $result;
+
+		$code = 0;
+		if (isset($result['error_code']) && is_int($result['error_code'])) {
```

Comment:
But that doesn't make any sense...? Why would you catch this exception, if there is nothing that throws it?